### ISSUE TITLE
About Image Entropy

### ISSUE THREAD

**Issue by 1e0nhardt** [CrossmodalGroup/DynamicVectorQuantization#7]
In [models/stage1_dynamic/dqvae_dual_entropy.py](https://github.com/CrossmodalGroup/DynamicVectorQuantization/blob/325801cda3ccf78cba633de49f4d0e1b513d3821/models/stage1_dynamic/dqvae_dual_entropy.py#L60C104-L60C104) . I think  `bins=torch.linspace(0, 1, 32)` should be changed to `bins=torch.linspace(-1, 1, 32)` , or do not use transforms.Normalize([0.5, 0.5, 0.5], [0.5, 0.5, 0.5] to preprocess images.

In Entropy.entropy(),
```python
values = values.unsqueeze(2)
residuals = values - bins.unsqueeze(0).unsqueeze(0)
kernel_values = torch.exp(-0.5 * (residuals / sigma).pow(2))

pdf = torch.mean(kernel_values, dim=1)
normalization = torch.sum(pdf, dim=1).unsqueeze(1) + epsilon
pdf = pdf / normalization + epsilon
```
When values which shape is [batch_size * patch_num, patch_pixel_nums] in range [-1, 1] and bin in range [0, 1], if all pixel values in patch is <0, then the residuals will be <0 too. As a result, kernel_values approaches 0 at an extremely fast rate when residuals decreasing and pdf will have several large value with other values close to 0, so the entropy value will be very low. 

This can lead to an issue where the entropy will be low regardless of the content if the brightness of the image is low. I think this is not the expected behavior.

**Comment by CrossmodalGroup** [comment]
Thanks for your interest and valuable advice! You are right and we find that this is benefical to change the bins to [-1,1]. We will update the corresponding code. We also update it in the readme.
